content/fr/docs/1.get-started/1.installation.md

```diff
@@ -15,6 +15,13 @@ Découvrez ci-dessous comment créer un projet Nuxt.js fonctionnel en 4 étapes.
 
 ---
 
+## Jouez avec Nuxt
+
+Vous pouvez jouer en ligne avec Nuxt sur CodeSandbox ou StackBlitz:
```

Comment:
"Jouer" in french does not work well. I would go with "Essayer"/"Tester" or something else